apps/fz_http/lib/fz_http/devices/device_setting.ex

```diff
@@ -11,14 +12,18 @@ defmodule FzHttp.Devices.DeviceSetting do
   embedded_schema do
     field :ip, :string
     field :ip6, :string
-    field :user_id, :integer
+    field :user_uuid, :integer
```

Comment:
You'll want to use `:string` or `Ecto.UUID` here I believe.

```suggestion
    field :user_uuid, Ecto.UUID
```